range_to_point_cloud_converter:
  angle_resolution: 0.045
  max_range: 2
